src/btree/bt_handle.c

```diff
@@ -948,23 +947,15 @@ __btree_page_sizes(WT_SESSION_IMPL *session)
      * item in an in-memory configuration.
      */
     if (F_ISSET(conn, WT_CONN_IN_MEMORY)) {
-        btree->maxintlkey = WT_BTREE_MAX_OBJECT_SIZE;
         btree->maxleafkey = WT_BTREE_MAX_OBJECT_SIZE;
         btree->maxleafvalue = WT_BTREE_MAX_OBJECT_SIZE;
         return (0);
     }
 
     /*
-     * In historic versions of WiredTiger, the maximum internal/leaf page key/value sizes were set
-     * by the internal_item_max and leaf_item_max configuration strings. Look for those strings if
-     * we don't find the newer ones.
+     * In historic versions of WiredTiger, the maximum leaf page key/value sizes were set by the
+     * leaf_item_max configuration string. Look for that string if we don't find the newer ones.
      */
-    WT_RET(__wt_config_gets(session, cfg, "internal_key_max", &cval));
-    btree->maxintlkey = (uint32_t)cval.val;
-    if (btree->maxintlkey == 0) {
-        WT_RET(__wt_config_gets(session, cfg, "internal_item_max", &cval));
-        btree->maxintlkey = (uint32_t)cval.val;
-    }
```

Comment:
Does this change mean we no longer need `internal_item_max`? If yes, we probably need to remove it from tests.

Reply:
This change doesn't have anything to do with `internal_item_max`, I don't think, it looks like that was removed some time ago, and references left in the tests. My guess is both `internal_item_max` and `leaf_item_max` should be removed from all occurrences, and that can be a new ticket. Nice catch!